menu_items/plugins.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+CURRENT_FILE="${BASH_SOURCE[0]}"
+CURRENT_DIR="$(dirname -- "$(readlink -f -- "$0")")"
+ROOT_DIR="$(dirname "$CURRENT_DIR")"
```

Comment:
this produces an error on my machine and is fixed if i set it to ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)"

Reply:
by this, I mean the ROOT_DIR